contracts/Vault.vy

```diff
@@ -1674,7 +1689,7 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
     # Update reporting time
     self.strategies[msg.sender].lastReport = block.timestamp
     self.lastReport = block.timestamp
-    self.lockedProfit = gain # profit is locked and gradually released per block
+    self.lockedProfit = gain * self.lockedProfitRatio / MAX_BPS  # profit is locked and gradually released per block
```

Comment:
return `total_fee` from `_assessFees` and modify this to:

```suggestion
    self.lockedProfit = gain - total_fee  # profit is locked and gradually released per block
```

NOTE: `total_fee` must be adjusted to not be greater than `gain`